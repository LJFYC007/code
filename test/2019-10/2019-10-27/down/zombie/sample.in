3
50 20
10 19 29350460 799341821
40 10 107685372 814201678
13 19 223889509 998804252
36 10 108134189 761714035
14 40 179913807 937335386
44 14 43456487 991621306
47 40 80524116 838958115
7 19 160051247 925155726
30 7 19842123 814109767
8 40 181977859 866415889
18 14 239882463 934146206
41 8 34533329 991889191
37 8 243786326 776532173
29 41 75120857 967655904
11 18 171005547 776458070
45 19 50097597 818823196
31 10 167010667 966593145
24 10 101244296 956982267
28 14 242529447 832724756
49 40 52241661 860057848
12 10 186069646 919753857
25 41 130218725 921227581
5 41 205574619 807790633
17 29 31904955 968513172
16 19 136743105 960047544
46 24 148291426 991125379
20 40 2544126 986086720
50 18 218516682 801673457
3 30 155048119 872554929
43 25 141986993 855578216
33 44 199887116 793092196
21 29 201015909 981707602
4 5 67749320 908244020
38 13 101693866 901834047
26 40 152989335 921074770
22 8 174469816 819191706
34 47 54398885 909212130
27 20 16300651 956992595
35 43 9582492 775598078
32 28 177360030 852915562
48 28 233762511 760153001
1 8 41596736 766236512
6 47 230587673 927612902
23 48 191540138 925621752
15 30 39146563 951065704
2 27 145970499 978536783
39 43 19830878 987227032
42 40 150721001 932471087
9 8 3981944 972237921
6 499551655
32 504114481
24 503347744
35 504556741
44 495932104
31 501996352
6 496472122
26 495655414
28 501429525
3 502666799
41 497072751
45 502310685
34 497253107
43 502499562
5 501622974
8 496644853
49 501180989
31 501254302
4 502756617
13 497860575
50 20
50 13 247506227 913934245
37 13 162170815 978633886
48 13 11976894 926388343
43 50 94399922 882384769
9 43 26007078 899063653
24 50 31236700 914484204
46 24 133920078 969717489
3 24 95400498 827363838
2 9 231781861 984205771
33 48 3279166 812409125
42 9 130671148 984966637
23 48 40976867 973215698
38 2 236413527 952396187
14 9 196408934 816583713
22 37 26126398 812752828
16 23 13965588 762290456
8 14 84355257 913166354
1 33 50853751 996557455
25 22 175304026 977072061
18 13 164055686 952395614
11 2 170381794 939862530
21 25 38093096 840012127
47 43 9378984 888318938
36 48 231513860 929208559
6 8 68728263 856620075
34 18 248974968 889694828
19 2 240635596 954423361
39 37 81783088 878963043
30 16 70716480 921361370
20 18 1974055 980414571
49 46 122221506 800565092
31 30 229738144 842910438
29 39 41469706 982804290
35 38 7508450 831492629
32 33 138234890 806990000
41 14 44826795 967363657
7 25 33839501 919730460
4 18 172908222 995915541
45 29 170679557 956745102
40 2 176876414 857928395
17 39 48821856 939484414
12 25 137497832 921843121
5 38 108295712 993983133
27 45 196946420 948858919
15 22 44360972 999444349
26 15 158415004 934646842
28 36 62837808 978723626
44 36 87633189 786058203
10 7 168075918 826312451
32 500195498
31 501387663
28 500089997
29 497085344
16 495904575
28 500426733
35 502532199
27 495137472
45 495713642
48 496781221
6 496249705
6 502089201
7 502263722
22 500683946
22 502600255
17 501455587
49 496442819
32 498175637
20 504340415
29 502887367
50 20
46 20 197686815 897018256
10 20 91179911 780753869
44 10 125856988 901595672
31 10 57803913 935842418
47 20 119827188 894606516
27 31 49769515 879062527
38 27 130204513 870390105
18 27 147124153 958296325
42 47 24288950 905443926
50 18 145548308 812168250
6 20 243944446 821996478
32 46 138681609 993863897
36 31 201137445 927255185
43 18 195016513 965296988
48 18 33678531 803111198
26 46 3560031 875004737
17 27 154980812 980372446
12 20 76499527 976565789
16 12 152093804 812846390
34 31 56051628 901645859
40 34 186536855 920130268
11 16 171306445 868013167
14 48 226262423 962366040
8 12 167012753 803893683
24 10 217243042 820575451
33 48 239317463 974877079
19 6 185972408 764538968
21 26 127562179 915489005
23 43 48275541 794734764
29 24 56592966 868374793
49 50 7763914 780945206
5 42 12769457 983996373
28 31 401967 782109225
2 18 3455645 785659782
39 20 15863112 815439881
3 48 194712036 877408880
25 36 26243680 844502917
30 33 201326982 893828469
13 17 113008092 959258928
45 31 249109416 952340736
22 49 184544155 989223506
35 25 205720598 828696579
7 47 117536216 849583417
1 29 6872299 945644545
9 42 239814443 880532821
37 8 46583501 821522044
15 17 12557667 933211407
4 16 71636323 848649445
41 23 43833156 945245836
28 496658207
12 499597891
39 496772838
21 495273060
21 504253234
2 496022043
34 496752975
40 504952595
8 495965149
25 500766628
22 503571802
34 502333467
23 498745495
24 497674561
5 504949034
48 501354970
26 503648953
47 498493381
44 500688759
12 496390731