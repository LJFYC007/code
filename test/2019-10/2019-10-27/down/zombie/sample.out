826089478
479227050
155430805
